primeqa/ir/dense/colbert_top/colbert/utils/parser.py

```diff
@@ -28,7 +28,7 @@ def add_model_parameters(self):
         self.add_argument('--doc_maxlen', dest='doc_maxlen', default=180, type=int)
 
         # Filtering-related Arguments
-        self.add_argument('--mask-punctuation', dest='mask_punctuation', default=False, action='store_true')
+        self.add_argument('--mask-punctuation', dest='mask_punctuation', default=True, action='store_true')
```

Comment:
How can you say not to mask punctuation (mask_punctuation = False)? With this statement it will always be True

Reply:
Good catch - the default behavior in `colbert/infra/config/settings.py` is to set `mask_punctuation=True` (https://github.com/primeqa/primeqa/blob/main/primeqa/ir/dense/colbert_top/colbert/infra/config/settings.py#L107), so to match this we could either remove this argument from the parser for now or add another flag like `--no-mask-punctuation` but also set its destination as `mask_punctuation` (the solution presented here: https://stackoverflow.com/a/15008806)
Is there a preference between these two? @franzmpub

Reply:
I pushed a commit with the latter fix but let me know if you'd prefer a different solution